content/episode/2017/09-13-109.md

```diff
@@ -0,0 +1,44 @@
++++
+date = "2017-09-13"
+draft = false
+title = "Выпуск 109. Обзор новостей и проектов"
+link = "golangshow.com/cdn/episodes/109.mp3"
+size = 17058725
+length = "00:40:46"
+categories = ["новости"]
++++
+
+## Темы
+
+- [Традиционный отчет от Саши](https://github.com/LK4D4/report/blob/master/reports/golang-09-13.md)
+  -  [testing: parallelize tests over count](https://github.com/golang/go/commit/f04d5836181dec3ec1b7e427607f02fa7a204a2d)
+  -  [strconv: optimize Atoi for common case](https://github.com/golang/go/commit/46aa9f5437b000fad3696b0cd9fd97995da16411)
+  -  [fmt: document verbs %b %d %o %x %X for printing pointers](https://github.com/golang/go/commit/a4140b745ce22c56821750001f30fca4020b4650)
+  - [cmd/fix: rewrite x/net/context by default](https://github.com/golang/go/commit/bf90da97c1aaec78d2f8ad8b74a506d3b6f0ee75)
+  -  [math/rand: add Shuffle](https://github.com/golang/go/commit/a2dfe5d278eae0864397a046a8206342a426d2bd)
+  - [runtime: improve timers scalability on multi-CPU systems](https://github.com/golang/go/commit/76f4fd8a5251b4f63ea14a3c1e2fe2e78eb74f81)
+  - [runtime: optimize siftupTimer and siftdownTimer a bit](https://github.com/golang/go/commit/6a7c63a08ab353c7e41cb24ae66e73fb3cb7cb56)
+- [Go does not inline functions when it should](https://lemire.me/blog/2017/09/05/go-does-not-inline-functions-when-it-should/)
+- [Community Outreach Working Group](https://blog.golang.org/community-outreach-working-group)
+- [NGINX Unit: A New Open Source App Server That Supports Go](https://www.nginx.com/products/nginx-unit/)
+- [rustgo: calling rust from go with near-zero overhead](https://blog.filippo.io/rustgo/)
+
+## Проекты, библиотеки и примеры кода
+
+- https://github.com/goby-lang/goby
+- https://github.com/posener/complete
+- https://transform.now.sh/json-to-go/
+- https://gnorm.org
+
+## Конференции и митапы
+
+* [DevFest Siberia 2017](https://gdg-siberia.com) в Новосибирске, 23-24 сентября
```

Comment:
Let's not mix `*` and `-` at least in the single document. I think we used `*` consistently before.

Reply:
Well, I always use `-` :) `*` is a copy-pasted part from the previous episode.
But I don't mind to use `*`, yeah.

Reply:
Fixed!